src/main/java/com/flippingutilities/utilities/SlotState.java

```diff
@@ -0,0 +1,34 @@
+package com.flippingutilities.utilities;
+
+import com.flippingutilities.model.OfferEvent;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import net.runelite.api.GrandExchangeOfferState;
+
+import java.time.Instant;
+
+@Data
+@AllArgsConstructor
+public class SlotState {
+    private boolean buy;
+    private int itemId;
+    private int currentQuantityInTrade;
+    private Instant time;
```

Comment:
How is this represented in JSON? Iso time string?

Reply:
a JSON object with two fields:

<img width="300" alt="Screen Shot 2021-08-31 at 11 28 35 PM" src="https://user-images.githubusercontent.com/21697386/131611941-861bf5e9-1dc6-49f8-abbe-21bc158910fb.png">

Reply:
The format should be an ISO instant, such as `2021-09-01T14:22:43.687Z`
Should be easy to do with the DateTimeFormatter